Codeinwp/neve
# fix: rtl support for react customizer control

### Summary
Adds back RTL support for react-based customizer controls and dashboard. This got broken while migrating to the new build process, but it should be ok now. :+1:  


### Will affect visual aspect of the product

NO

### Test instructions


- Install RTL Tester.
- Check the Customizer Controls and Dashboard page. Everything should look as expected.


Closes ##2086.
